package-release.sh

```diff
@@ -8,15 +8,17 @@ fi
 DXVK_VERSION="$1"
 DXVK_SRC_DIR=`dirname $(readlink -f $0)`
 DXVK_TMP_DIR="/tmp/dxvk-$DXVK_VERSION"
-DXVK_ARCHIVE_PATH="$2/dxvk-$DXVK_VERSION.tar.gz"
+DXVK_ARCHIVE_PATH=$(realpath "$2")"/dxvk-$DXVK_VERSION.tar.gz"
 
 function build_arch {
   cd "$DXVK_SRC_DIR"
 
   meson --cross-file "$DXVK_SRC_DIR/build-win$1.txt"  \
         --buildtype "release"                         \
         --prefix "$DXVK_TMP_DIR/install.$1"           \
+	--unity on                                    \
```

Comment:
I'd like to do more extensive testing before enabling this by default, and have time to fix potential problems.

People may (well, some actually **do**) use the script to build DXVK, no reason to potentially break it for them.

Reply:
I agree, although in theory there shouldn't be issues with it from what I've read.